### DISCUSSION TITLE
Ability to be able to add every protection enchants at once.

### DISCUSSION THREAD

**Discussion by GamehunterKaan** [PurpurMC/Purpur#439] [__Completed__] [answered]
There should be an option for adding every single protection enchants on survival.This is already can be done by commands but being able to do this with enchanted books in survival would be really good feature.

**Comment by BillyGalbreath** [comment, marked as answer]
Implemented in https://github.com/pl3xgaming/Purpur/commit/58c99ab6d1806ec68f965c0aafc6aba0f4f4e153
